### PR TITLE
enable color in log on Windows

### PR DESCRIPTION
**Pull request by mattn**
See https://github.com/lileio/lile/issues/35

### CODE REVIEW

**Comment by mattn** [comment]
will fix conflict

File: template/cmd_main.tmpl [hunk 1]
```diff
@@ -1,29 +1,39 @@
...
-        "{{ .RelDir }}/server"
+	"github.com/mattn/go-colorable"
+	"github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+	"{{ .RelDir }}"
+	"{{ .RelDir }}/server"
 	"{{ .RelDir }}/{{ .Name }}/cmd"
 )
 
+func init() {
```
Comment (arbarlow) [inline template/cmd_main.tmpl]: Since logrus is global we'll only need to do this once, in the above commit is therefor fine!

File: lile.go [hunk 2]
```diff
@@ -63,6 +67,12 @@ type Service struct {
 
 // NewService creates a new service with a given name
 func NewService(n string) *Service {
+	once.Do(func() {
```
Comment (arbarlow) [inline lile.go]: What about just moving this to an init? We actually have a `lile_windows.go` now

**Comment by arbarlow** [comment]
Closing due to inactivity, but I'm happy to re-open and discuss support for windows!

File: lile.go (hunk 2 above)
Comment (mattn) [reply lile.go]: moving this to init doesn't work since some packages refer this package on start-up.

**Pull request closed without merging**